spec/models/well_spec.rb

```diff
@@ -261,31 +261,46 @@
-    [10, 100, 50, 1, 9, nil],
-    [1000, 250, 50, 4, 6, nil],
-    [10_000, 250, 50, 40, 0, nil],
-    [10_000, 250, 30, 30, 0, nil]
-    # rubocop:todo Metrics/ParameterLists
-  ].each do |target_ng, measured_concentration, measured_volume, stock_to_pick, buffer_added, current_volume|
-    # rubocop:enable Metrics/ParameterLists
+    { target_ng: 1000, measured_conc: 10, measured_vol: 50, min_vol:10, min_pick_vol: 1, source_pick_vol: 50, buffer_vol: 0, current_vol: nil },
+    { target_ng: 1000, measured_conc: 10, measured_vol: 10, min_vol:10, min_pick_vol: 1, source_pick_vol: 10, buffer_vol: 0, current_vol: nil },
+    { target_ng: 1000, measured_conc: 10, measured_vol: 20, min_vol:10, min_pick_vol: 1, source_pick_vol: 10, buffer_vol: 0, current_vol: 10 },
+    { target_ng: 100, measured_conc: 100, measured_vol: 50, min_vol:10, min_pick_vol: 1, source_pick_vol: 1, buffer_vol: 9, current_vol: nil },
+    { target_ng: 1000, measured_conc: 1000, measured_vol: 50, min_vol:10, min_pick_vol: 1, source_pick_vol: 1, buffer_vol: 9, current_vol: nil },
+    { target_ng: 5000, measured_conc: 1000, measured_vol: 50, min_vol:10, min_pick_vol: 1, source_pick_vol: 5, buffer_vol: 5, current_vol: nil },
+    { target_ng: 10, measured_conc: 100, measured_vol: 50, min_vol:10, min_pick_vol: 1, source_pick_vol: 1, buffer_vol: 9, current_vol: nil },
+    { target_ng: 1000, measured_conc: 250, measured_vol: 50, min_vol:10, min_pick_vol: 1, source_pick_vol: 4, buffer_vol: 6, current_vol: nil },
+    { target_ng: 10_000, measured_conc: 250, measured_vol: 50, min_vol:10, min_pick_vol: 1, source_pick_vol: 40, buffer_vol: 0, current_vol: nil },
+    { target_ng: 10_000, measured_conc: 250, measured_vol: 30, min_vol:10, min_pick_vol: 1, source_pick_vol: 30, buffer_vol: 0, current_vol: nil },
+    { target_ng: 1000, measured_conc: 70, measured_vol: 50, min_vol:10, min_pick_vol: 5, source_pick_vol: 14.29, buffer_vol: 0, current_vol: nil },# Y24-382: SQPD-10861 v14.29, b0.00
+    { target_ng: 200, measured_conc: 200, measured_vol: 1, min_vol:50, min_pick_vol: 5, source_pick_vol: 1, buffer_vol: 49, current_vol: nil }, # Y24-382: SQPD-10864 v1.00, b49.00
+    { target_ng: 9800, measured_conc: 98, measured_vol: 100, min_vol:50, min_pick_vol: 5, source_pick_vol: 50, buffer_vol: 0, current_vol: nil }, # Y24-382: SQPD-10866 v50.00, b0.00
+    { target_ng: 9800, measured_conc: 100, measured_vol: 100, min_vol:50, min_pick_vol: 5, source_pick_vol: 50, buffer_vol: 0, current_vol: nil } # Y24-382: SQPD-10868 v50.00, b0.00
+  ].each do |cherrypick|
+    # rubocop:enable Layout/LineLength
+    target_ng = cherrypick[:target_ng]
+    concentration = cherrypick[:measured_conc]
+    measured_volume = cherrypick[:measured_vol]
+    stock_to_pick = cherrypick[:source_pick_vol]
+    buffer_added = cherrypick[:buffer_vol]
+    current_volume = cherrypick[:current_vol]
+    minimum_volume = cherrypick[:min_vol]
+    robot_minimum_picking_volume = cherrypick[:min_pick_vol]
+
```

Comment:
You haven't added your v10.71, b5.00 example, which generated some questions and discussion.